File: app/src/main/java/com/mobile/rxjava2andretrofit2/login/view/ILoginView.java
package com.mobile.rxjava2andretrofit2.login.view;

import com.mobile.common_library.base.IBaseView;
import com.mobile.rxjava2andretrofit2.login.bean.LoginResponse;

public interface ILoginView extends IBaseView {

    void loginSuccess(LoginResponse.DataBean success);

    void loginError(String error);

}

File: library_common/src/main/java/com/phone/library_common/login/view/ILoginView.kt
package com.phone.library_common.login.view

import com.phone.library_common.base.IBaseView
import com.phone.library_common.login.bean.DataGetVerification
import com.phone.library_common.login.bean.DataLogin

interface ILoginView : IBaseView {

    fun getAuthCodeSuccess(success: DataGetVerification)

    fun getAuthCodeError(error: String)

    fun loginWithAuthCodeSuccess(success: DataLogin)

    fun loginWithAuthCodeError(error: String)

    fun loginSuccess(success: String)

    fun loginError(error: String)

}